statsmodels/tsa/holtwinters.py

```diff
@@ -0,0 +1,444 @@
+"""
+Created on Wed Jul 12 09:35:35 2017
+
```

Comment:
You should add your full name.  Also any references would be appreciated.

Reply:
Done